packages/client/hmi-client/src/components/workflow/ops/simulate-ciemss/tera-simulate-node-ciemss.vue

```diff
@@ -146,6 +149,7 @@ const preparedCharts = computed(() => {
 	Object.keys(pyciemssMap).forEach((key) => {
 		reverseMap[`${pyciemssMap[key]}_mean`] = key;
 	});
+	const xAxisTitle = model.value?.semantics?.ode.time.units?.expression;
```

Comment:
I think this should be computed since the `model` could change. Maybe not needed in the drilldown

Reply:
Noted, but I think that is going to be a general problem with our hand-wavy way of updating things.

The node-component has no means of knowing when the model itself has changed, when all it has is an model-configuration-id (not even a model-id).